script_tests/tests/1_test-transfer.js

```diff
@@ -0,0 +1,86 @@
+const expect = require("chai").expect;
```

Comment:
Why rename these test files name to `1_test-transfer.js `, `2_test-constract-log.js `, `3_test-rpc-filter.js`, `4_test-solidity-opcodes.js `?

Reply:
It defines the test order.